openvdb/CMakeLists.txt

```diff
@@ -215,7 +215,25 @@ SET_TARGET_PROPERTIES(
   VERSION ${OPENVDB_MAJOR_VERSION_NUMBER}.${OPENVDB_MINOR_VERSION_NUMBER}.${OPENVDB_PATCH_VERSION_NUMBER}
   )
 
-SET ( VDB_PRINT_SOURCE_FILES  cmd/openvdb_print/main.cc )
+SET ( VDB_LOD_SOURCE_FILES  cmd/openvdb_lod.cc )
+SET_SOURCE_FILES_PROPERTIES ( ${VDB_LOD_SOURCE_FILES}
+  PROPERTIES
+  COMPILE_FLAGS "-DOPENVDB_USE_BLOSC"
+  )
```

Comment:
Correct me if I'm wrong, but OPENVDB_USE_BLOSC is purely for the objects built by the core library and isn't required anymore for dependant builds (except for the write SOP which uses it to introduce a blosc mode)? Or is that just happenstance and we want to continue to make sure people are defining this in-case it makes its way back into a header

Reply:
You're not wrong, but all of the other executables get that flag, so it's possible that CMake requires it to be set at that level.  Then again, all of the other executables get `OPENVDB_USE_GLFW_FLAG`, which is needed only when building the viewer, so maybe not.

Reply:
I'm fairly certain it's not needed, but I don't mind it staying for now to match the rest of the CMake until it's cleaned up